kernel/data-pipeline/dialect/opengauss/src/main/java/org/apache/shardingsphere/data/pipeline/opengauss/ingest/wal/decode/MppdbDecodingPlugin.java

```diff
@@ -57,9 +57,7 @@ public final class MppdbDecodingPlugin implements DecodingPlugin {
     
     private final boolean decodeWithTX;
     
-    public MppdbDecodingPlugin(final BaseTimestampUtils timestampUtils) {
-        this(timestampUtils, false);
-    }
+    private final int majorVersion;
```

Comment:
Could we replace `int majorVersion` to `boolean decodeParallelly`, it's error-prone to to transfer majorVersion (e.g. 2, 3 or any other integer) on invocation.